recipes/momlevel/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "momlevel" %}
+{% set version = "0.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/momlevel-{{ version }}.tar.gz
+  sha256: e84390f2f588f221203e6837c11e8141f3d5396bb65644226906b041d4d6c5fc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - importlib-metadata <5.0.0
+    - nc-time-axis
+    - netcdf4
+    - numpy >=1.21.0
+    - scikit-learn >=1.0.1
+    - xarray >=0.20.0
+    - xgcm
+
+test:
+  imports:
+    - momlevel
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jkrasting/momlevel
+  summary: Tools to diagnose sea level in the MOM Ocean Model
+  description: |
+    momlevel is a Python package in the public domain that provides
+    documentation and routines for analyzing sea level and related
+    quantities in MOM6-class models. The package initially focuses
+    on global and local steric sea level change in Boussinesq configurations.
+  license: Public Domain
```

Comment:
This license is not allowed, as mentioned by the linker. If it is not a standard license, please create a custom License Ref: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions